SwiftUI/GraphicsHub/GraphicsHub/Renderer/SpecializedRenderers/Tests/TestRenderers.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  TestRenderers.swift
+//  GraphicsHub
+//
+//  Created by Noah Pikielny on 9/7/21.
+//
+
+import MetalKit
+
+class UVRenderer: Renderer {
```

Comment:
All your other classes have spaces in between class header and body.  Keep format consistent